.github/workflows/build-examples-bzlmod.yml

```diff
@@ -0,0 +1,28 @@
+name: Build examples/bzlmod
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+
+jobs:
+  build:
+    name: ${{ matrix.name }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - name: linux-amd64
+            os: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Install Bazelisk
+        uses: bazelbuild/setup-bazelisk@v3
```

Comment:
I suspect this is not needed because github action has bazelisk installed out of the box